src/test/kotlin/org/opensearch/commons/alerting/model/DocLevelMonitorInputTests.kt

```diff
@@ -44,6 +44,20 @@ class DocLevelMonitorInputTests {
     @Test
     fun `test create Doc Level Query with invalid name length`() {
         val stringBuilder = StringBuilder()
+
+        // test empty string
+        val emptyString = stringBuilder.toString()
+        try {
+            randomDocLevelQuery(name = emptyString)
+            Assertions.fail("Expecting an illegal argument exception")
+        } catch (e: IllegalArgumentException) {
+            Assertions.assertEquals(
+                "The query name, $emptyString, can be max 256 characters.",
```

Comment:
Minor: the error message should say between 1-256 characters

Reply:
done